nexus/media/js/lib/facebox/facebox.js

```diff
@@ -81,19 +81,22 @@
    * Public, $.facebox methods
    */
 
+  var nexusMediaPreix = $('#nexus-facebox-constants').attr('data-nexus-media-prefix'),
+      closeImage = nexusMediaPrefix + 'nexus/img/facebox/closelabel.gif';
+
   $.extend($.facebox, {
```

Comment:
eventually this should be moved into nexus's own JS so you can update facebox.js to 1.3 from https://github.com/defunkt/facebox

Sadly this version of 1.2 and the upstream version have significant changes